CHANGES.rst

```diff
@@ -1,5 +1,13 @@
 .. currentmodule:: werkzeug
 
+Version 3.1.0
+-------------
+
+Unreleased
+
+-  Support Cookie CHIPS (Partitioned Cookies). :pr:`2855`
```

Comment:
Can link to the issue https://github.com/pallets/werkzeug/issues/2797

Reply:
Yep